src/super_gradients/training/utils/quantization_utils.py

```diff
@@ -0,0 +1,384 @@
+"""
+Quantization utilities
+
+Methods are based on:
+ https://github.com/NVIDIA/TensorRT/blob/51a4297753d3e12d0eed864be52400f429a6a94c/tools/pytorch-quantization/examples/torchvision/classification_flow.py#L385
+
+(Licensed under the Apache License, Version 2.0)
+"""
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+import torch
+from super_gradients.common.abstractions.abstract_logger import get_logger
+from super_gradients.training.utils.callbacks import Phase, PhaseCallback, PhaseContext
+import os
+from enum import Enum
+from super_gradients.training.utils.checkpoint_utils import load_checkpoint_to_model
+from super_gradients.training.utils import get_param
+from super_gradients.training.utils.distributed_training_utils import get_local_rank, \
+    get_world_size
+from torch.distributed import all_gather
+
+logger = get_logger(__name__)
+
+try:
+    _imported_trt_failure = None
```

Comment:
not sure I understand what is going on here...

Reply:
Redundant trt import we used for removed trt functions. removed in https://github.com/Deci-AI/super-gradients/pull/232/commits/2a47695a2aa92b6c80bfc4952528312906d655ae